package.json

```diff
@@ -1975,30 +2035,77 @@
 					"command": "issue.copyGithubHeadLink",
 					"when": "github:hasGitHubRemotes",
 					"group": "1_githubPullRequests@3"
+				},
+				{
+					"command": "issue.copyGithubDevLinkFile",
+					"when": "github:hasGitHubRemotes && remoteName == codespaces || github:hasGitHubRemotes && embedderIdentifier == github.dev",
+					"group": "0_vscode@0"
 				}
 			],
-			"editor/title/context": [
+			"editor/lineNumber/context": [
 				{
 					"command": "issue.copyGithubPermalink",
 					"when": "github:hasGitHubRemotes",
-					"group": "1_cutcopypaste@10"
+					"group": "1_cutcopypaste@1"
```

Comment:
I inherited this group name from `editor/title/context` so that we can show the Copy vscode.dev Link action in the same group as the GHPRI actions (so that we do not have a single copy action in its own group in the `editor/lineNumber/context` menu). Thanks for asking--I realize I put the Copy github.dev action into the wrong group (should also be `1_cutcopypaste`). The other ones can be renamed to `1_githubPullRequests`.